building.html

```diff
@@ -0,0 +1,193 @@
+          <li><a href="https://openal-soft.org/">OpenAL Soft</a></li>
+          <li><a href="https://libexpat.github.io/">Expat</a></li>
+          <li><a href="https://libjpeg-turbo.org/">libjpeg</a></li>
+          <li><a href="http://www.libpng.org/">libpng</a></li>
+          <li><a href="https://zlib.net/">zlib</a></li>
+          <li><a href="https://curl.se/">libcurl</a></li>
+          <li><a href="https://github.com/zrax/string_theory/">string_theory</a></li>
+          <li><a href="https://think-async.com/Asio/">Standalone ASIO</a></li>
+        </ul>
+
+        <p>The following libraries are optional:</p>
+        <ul>
+          <li><a href="https://cairosvg.org/">CairoSVG</a> (for building resource.dat)</li>
+          <li><a href="https://python-pillow.org/">Pillow</a> (for building resource.dat)</li>
+          <li><a href="https://www.freetype.org/">FreeType</a> (for plFontConverter)</li>
+          <li><a href="https://www.qt.io/download-open-source">Qt5</a> (for the GUI tools)</li>
+          <li><a href="https://www.webmproject.org/">VPX and WebM</a> (for video)</li>
+          <li><a href="https://www.opus-codec.org/">Opus</a> (for video and voice chat)</li>
+          <li><a href="https://www.speex.org/downloads/">Speex</a> (for legacy voice chat)</li>
+          <li>Autodesk 3ds Max 7 (non-free SDK; for 3ds Max plugin)</li>
+        </ul>
+      </details>
+
+
+      <h3 id="building-windows">Building on Windows</h3>
+
+      <p>Compiling on Windows requires Visual Studio and the Windows SDK. We recommend <a href="https://visualstudio.microsoft.com/downloads/">Visual Studio 2019 Community Edition</a>, which is available as a free download. During the installation of Visual Studio, the Windows SDK can be installed by selecting the “Desktop development with C++” workload.</p>
+
+      <ol>
+        <li>
+          <p>Clone the repository from GitHub, including all submodules, using a git client. On the command line, this can be done by executing the following:<br>
+            <code>git clone --recurse-submodules https://github.com/H-uru/Plasma.git</code></p>
+
+          <p class="infonote" data-infotype="warning"><b>Note:</b> We recommend using <a href="https://git-scm.com/download/win">Git for Windows</a>. Using the git program provided by Cygwin will not set permissions properly on some files and result in build failures.</p>
+
+          <p class="infonote" data-infotype="warning"><b>Note:</b> Some dependencies may fail to build if the path to the code contains spaces.</p>
+        </li>
+
+        <li>
+          <p>Launch Visual Studio.</p>
+        </li>
+
+        <li>
+          <p>Select <strong>Open a local folder</strong> and choose the folder where you cloned the repository.</p>
+        </li>
+
+        <li>
+          <p>Compile the project by using <strong>Build &gt; Install Plasma</strong>. All required dependencies will automatically be downloaded from vcpkg as part of the build process.</p>
```

Comment:
This will build a debug client.

Reply:
Is that what we want the default to be? Can we change the default by attempting to set a default `CMAKE_BUILD_TYPE`?